docker/setLogLevel.xsl

```diff
@@ -11,7 +11,7 @@
             <xsl:apply-templates select="node()|@*"/>
                 <log:logger category="org.keycloak">
                     <log:level>
-                      <xsl:attribute name="name">${env.KEYCLOAK_LOGLEVEL:INFO}</xsl:attribute>
```

Comment:
did nothing! :-/ I believe it was bc of https://github.com/almighty/keycloak/pull/17/files#diff-9f3d734a0b7662c1a6181c1795c40024R111